sys/Makefile

```diff
@@ -121,6 +121,9 @@ endif
 ifneq (,$(filter l2filter,$(USEMODULE)))
     DIRS += net/link_layer/l2filter
 endif
+ifneq (,$(filter memarray,$(USEMODULE)))
+    DIRS += memarray
```

Comment:
Indent should be 2 spaces here (RIOT style for Makefiles)

Reply:
fixed, thanks